src/variety/fillomino.js

```diff
@@ -410,6 +428,10 @@
 		bsSameNum: [
 			"同じ数字のブロックが辺を共有しています。",
 			"Adjacent blocks have the same number."
+		],
+		bkNotSymmRoom: [
+			"部屋の形が点対称ではありません。",
+			"The room is not point symmetric."
```

Comment:
I think they're called "blocks" here. If we were to change that, I think I'd prefer "area". "room" sounds like something given for me.

So the small change would be: `bkNotSymm` ... "The block is not..."

Reply:
Fair enough. I didn't see any consistency there already (cf. line 424), so I wasn't sure what word to use. Using "block" is fine by me.